_data/sites.json

```diff
@@ -7882,6 +7882,15 @@
         ]
     },
 
+    {
+        "name": "Rail Europe",
+        "url": "https://www.raileurope.com/account/settings",
+        "difficulty": "easy",
+        "notes": "Go to account settings and click the 'Delete my account' button at the top right corner",
```

Comment:
```suggestion
        "difficulty": "hard",
        "notes": "Go to account settings and click the 'Delete my account' button at the top right corner. This will open a ticket for them to delete your account and it will eventually be processed.",
```